quickstart/csharp/dotnet/intent-recognition/README.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart: Recognize speech in C# under .NET Framework for Windows
```

Comment:
>Recognize speech [](start = 14, length = 16)

This is the same document as under quickstart/csharp/dotnet/from-microphone. Should probably be specified for intent recognition.

Reply:
Please triage and find a home for this WI.

[https://msasg.visualstudio.com/Skyman/_workitems/edit/2172956](https://msasg.visualstudio.com/Skyman/_workitems/edit/2172956)


---
In reply to: [344385800](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/429#discussion_r344385800) [](ancestors = 344385800)